lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -740,67 +740,57 @@ verify_check => md5_hex($check->test_query),
 
     my $rows = [];
     while (my $row = $sth->fetchrow_hashref('NAME_lc')) {
-      my $id = $row->{$check->{id_column}};
+      my $count = 1+scalar(@$rows);
+      #TODO: A '' will fail with invalid input syntax in fix_tests - YL
```

Comment:
When dealing with multiple keys, we may find cases when not all are present in user record. We should deal with that.